src/testrel.jl

```diff
@@ -26,11 +26,13 @@ function set_context(new_context::Context)
     return TEST_CONTEXT_WRAPPER.context = new_context
 end
 
-function create_test_database(clone_db::Union{Nothing, String} = nothing)::String
+function create_test_database_name()::String
     basename = get(ENV, "TEST_REL_DB_BASENAME", "test_rel")
```

Comment:
```suggestion
function create_test_database_name(; default_basename="test_rel")::String
    basename = get(ENV, "TEST_REL_DB_BASENAME", default_basename)
```
just to future-proof it a bit for when this more widely used as a package